iterate_dynamodb/datasources/dynamodb_ci.datasource

```diff
@@ -0,0 +1,16 @@
+
+SCHEMA >
+    `pk` String `json:$.pk`,
+    `sk` String `json:$.sk`,
+    `slug` String `json:$.slug` DEFAULT 'x',
+    `now_ts` DateTime64(3) `json:$.now_ts` DEFAULT now64()
+
+ENGINE "ReplacingMergeTree"
+ENGINE_PARTITION_KEY "toYYYYMM(toDateTime64(_timestamp, 3))"
+ENGINE_SORTING_KEY "pk, sk"
+ENGINE_VER "_timestamp"
```

Comment:
Not sure what we do in these cases, but these fields are not mandatory here. These are the default ones.

Users should only be adding them in case of overriding some (and knowing what they do 😓)

Reply:
Maybe we can add a comment here in the Datafile like "this is the default setting and you don't need to write it but you could override with your own settings..."